iocore/net/SSLCertLookup.cc

```diff
@@ -160,7 +160,40 @@ ticket_block_alloc(unsigned count)
 
   return ptr;
 }
+ssl_ticket_key_block *
+ssl_create_ticket_key_block_buffer(char *ticket_key_data, int ticket_key_len)
+{
+  ssl_ticket_key_block *keyblock = NULL;
+  int num_ticket_keys            = ticket_key_len / sizeof(ssl_ticket_key_t);
+  if (num_ticket_keys == 0) {
+    Error("SSL session ticket key is too short (>= 48 bytes are required)");
+    goto fail;
+  }
+
+  // Increase the stats.
+  if (ssl_rsb != NULL) { // ssl_rsb is not initialized during the first run.
+    SSL_INCREMENT_DYN_STAT(ssl_total_ticket_keys_renewed_stat);
+  }
```

Comment:
Isn't the point of this metric to track the number of times new ticket keys are loaded?  Not sure I'm following your side-effect comment.